### PR TITLE
Added support for Bower packages (resolves #65)

### LINKED ISSUE #65: Add support for bower packages
It would be great if postcss-import natively supported import of bower packages the same way it supports npm modules.

For example, it seems more logic to me to have [normalize.css](https://github.com/necolas/normalize.css) installed through Bower than through npm.

Thanks!

### PR DESCRIPTION
**Pull request by pensierinmusica**
I updated `index.js` and `README.md`. I didn't add any new tests though, since it looks like the conditions already tested for `node_modules` perfectly satisfy `bower_components` too.

Cheers!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,57 +84,57 @@ Checkout [tests](test) for more examples.
 
 #### `root`
 
-Type: `String`
```
Comment (MoOx) [inline README.md]: The 2 spaces are intentional (to create new lines). Can you put them back ? Using editorconfig might helps to respect that (see https://github.com/postcss/postcss-import/blob/master/.editorconfig#L12-L13 )

File: index.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ var resolvedPromise = new Promise(function(resolvePromise) {
 var moduleDirectories = [
   "web_modules",
   "node_modules",
+  "bower_components"
```
Comment (MoOx) [inline index.js]: You forgot the trailing coma :)

**Comment by pensierinmusica** [comment]
Done! Sorry for the confusion, it took me a bit to find out exactly where the spaces originally where. Thanks for the suggestion to use editorconfig, seems like a good idea! Cheers

**Comment by MoOx** [comment]
One more thing, can you squash your commits into one please? Thanks!

**Comment by pensierinmusica** [comment]
Sure, makes sense  :)  Done!

**Comment by MoOx** [comment]
Released as 6.1.0

**Comment by pensierinmusica** [comment]
Wonderful  :)

**Pull request merged**